### ISSUE TITLE
Enchantment mod compatability 

### ISSUE THREAD

**Issue by GanerCodes** [Sephta/stal-alloys#5]
Hi, I'm unsure why this is occurring but mods that alter the max ench. levels don't work on some items in this pack. i.e. putting protection V on stainless steel armor. Here's the mod in question I use, and I don't think it is at fault as it doesn't seem to have some sort of list of armor/tools it modifies.
https://github.com/mastery3/sharpness6

**Comment by Sephta** [comment]
Hello, I did a bit of testing on my end and it seems that enchanting doesnt work at all with the items of my mod. I'll have to look into why this is. I'll try and put out a patch sometime soon but not sure when just yet. Thank you for reporting this issue.

**Comment by GanerCodes** [comment]
thanks; also similar story when it comes to this mod https://github.com/craftycorvid/MergedElytra

if i had to guess it may be some issue with tagging?

**Comment by Sephta** [comment]
Yeah I think you're right. I saw some people having similar issues. Should be an easy fix, and I can take a look at it sometime this week.

**Comment by Sephta** [comment]
This is definitely a tag issue. I'm currently working on a fix and will have a release this week

**Comment by Sephta** [comment]
I've pushed some changes and generated a new jar file. Ill be pushing an update for Minecraft 1.21 soon. 

@GanerCodes what version of Minecraft do you use with my mod? I can prioritize that version next if you'd like.

**Comment by GanerCodes** [comment]
1.21, thanks!

**Comment by Sephta** [comment]
Issue resolved as of update 1.2.3 for Minecraft 1.21 and 1.21.1
[Modrinth Link](https://modrinth.com/mod/stal-alloys/version/hkUgZzFE)
[Curseforge Link](https://www.curseforge.com/minecraft/mc-mods/stal-alloys/files/all?page=1&pageSize=20)
